### ISSUE TITLE
Trails under 1 mi showing in POI Trail Access info

### ISSUE THREAD

**Issue by versatilehuman** [smartchicago/trailsy#89]
Related to #88?
 
![image](https://cloud.githubusercontent.com/assets/7032169/16702881/93c9ca68-452e-11e6-9ca4-b6119059d380.png)

**Comment by shua123** [comment]
Yes this is related to the other issue. With the earlier data, I thought the POI-Trail connections were just the trail segments closest (or connected) to the POI. So the way this was set up is that the top showed the direct connection trails and then we had the "WITH ACCESS TO" park that displayed the rest of the trail color-type sections of the trail subsystem. For the under 1 mile filter, I only applied that to the indirect trails (the ones below "with access to." If it was a direct connection, I showed them regardless of length. There were cases where if I filtered by length with the direct connections, there would be nothing about the "with access to.' I also thought it was confusing to see a trail color coming right from the trailhead/poi and not have it acknowledged (the green dashed part at Beck Lake for example).
![image](https://cloud.githubusercontent.com/assets/1593537/16709834/e793fa8e-45e1-11e6-8f77-aac5732b6860.png)
